tutorials/multimodal_dapt_curation/curator/main.py

```diff
@@ -420,7 +421,15 @@ def main() -> None:
     image_ddf = process_image_data(data_type_map)
 
     # Initialize the client and run the curation pipelines
-    client = get_client(**ArgumentHelper.parse_client_args(args), set_torch_to_use_rmm=True)
```

Comment:
can you try running with `get_client` itself along with the `CUDF_SPILL` env variable.. make sure you're passing the same thing to your `args` of 1gb / tcp / etc